app/controllers/account/companies_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
     @company = current_user.companies.build(company_params)
     FullContactCompanyProcessor.new(company: @company).process
     if @company.save
+      BingApiV7.new(@company).pages_process
```

Comment:
move this to worker - NewCompanyWorker
search is taking some time and user should not wait